src/cmd/serve.rs

```diff
@@ -27,6 +27,9 @@ pub struct Serve {
     /// The port to serve on [default: 8080]
     #[arg(long, env = "TRUNK_SERVE_PORT")]
     pub port: Option<u16>,
+    /// The aliases to serve on
+    #[arg(short, long, env = "TRUNK_SERVE_ALIAS")]
+    pub aliases: Option<Vec<String>>,
```

Comment:
I think this should be the singular, just like `address`, so that you can use `--alias foo`. Which also aligns this with the env-var variant.

As the CI complains, the `-a` argument is already taken. As is the `-A`. I'd say just drop the short version for this one.